Firestore/Example/Podfile

```diff
@@ -11,26 +11,54 @@ pod 'FirebaseFirestore', :path => '../../'
 
 target 'Firestore_Example_iOS' do
   platform :ios, '8.0'
+end
```

Comment:
A comment here to look below for construction of the dependent targets could be helpful.